docs/modules/ROOT/pages/how-to-guides/proc_maintaining_operator_graphs.adoc

```diff
@@ -0,0 +1,222 @@
+:_content-type: PROCEDURE
+:myfunctionone: maintaining_operator_graphs
+
+[id="maintaining_operator_graphs_{context}"]
+= Maintaining operator graphs using the information on the File-based catalogs
```

Comment:
Could the title be simplified? Would `maintaining operator graphs using file-based catalogs` be as good? Does it need to call out FBC, or could that be the mechanism that RHTAP uses and describes in the doc? Is "maintaining an operator graph" meaningful to all operator developers?